axis.go

```diff
@@ -497,6 +498,158 @@ func (utt UnixTimeTicks) Ticks(min, max float64) []Tick {
 	return ticks
 }
 
+// AutoUnixTimeTicks is suitable for axes representing time values.
+// AutoUnixTimeTicks expects values in Unix time seconds.  It will
+// adjust the number of ticks according to the specified Width.  If
+// not specified, Width defaults to 4 inches.
+type AutoUnixTimeTicks struct {
```

Comment:
heh.. I know it isn't pretty :) `UnixTimeTicks` was already taken.. and `Auto` was used to distinguish the added feature..

Could we make it `TimeTicks`  ? and document the fact that you need Unix timestamps, or provide a function to transform a `[]time.Time` into whatever this thing needs ?